src/components/pages/Profile.tsx

```diff
@@ -615,6 +594,43 @@ export const Profile = ({ handle }: { handle?: string | null }) => {
           />
         }
       />
+      <HoverToggle
+        fadeOut={
+          <LogoCard
+            title="Disco"
+            logo={"/disco.svg"}
+            logoClassName={"scale-110 invert"}
+          />
+        }
+        fadeIn={
+          <AppCard
+            onClickLogo={openInNewTab("https://disco.xyz")}
+            title="Disco"
+            logo="/disco.svg"
+            logoAlt="Disco Logo"
+            onClickLinkOut={openInNewTab("https://disco.xyz/")}
+            onClickSendMessage={onClickSendMessage()}
+            linkOutText={"View on Disco"}
+            logoClassName="invert"
+          />
+        }
+      />
+      <HoverToggle
+        fadeOut={<LogoCard title="PolygonId" logo={"/polygonid.svg"} />}
```

Comment:
The display string for this should have a space between "polygon" and "id". See 
<img width="829" alt="image" src="https://user-images.githubusercontent.com/100799178/197078835-e3fb207d-3537-48a9-8a0f-0bc669ea8195.png">